etcdctl/README.md

```diff
@@ -165,6 +165,40 @@ The protobuf encoding of the DeleteRange [RPC response][etcdrpc].
 ./etcdctl get foo
 ```
 
+```bash
+./etcdctl put key val
+# OK
+./etcdctl del --prev-kv key
+# 1
+# key
+# val
+./etcdctl get key
+```
+
+```bash
+./etcdctl put foo1 bar
+# OK
+./etcdctl put foo2 bar
+# OK
+./etcdctl put foo3 bar
+# OK
+./etcdctl del --from-key foo
+# 3
+./etcdctl get foo3
```

Comment:
this example would work the same if given `--prefix` instead of `--from-key`, how about
```bash
./etcdctl put a 123
# OK
./etcdctl put b 456
# OK
./etcdctl put z 789
# OK
./etcdctl del --from-key a
# 3
./etcdctl get --from-key a
```